package.json

```diff
@@ -24,7 +24,7 @@
     "@sanity/desk-tool": "^2.12.2",
     "@sanity/production-preview": "^2.2.6",
     "@sanity/vision": "^2.11.2",
-    "@webriq-pagebuilder/sanity-plugin-schema-default": "^0.0.4",
+    "@webriq-pagebuilder/sanity-plugin-schema-default": "^0.0.6",
```

Comment:
`@webriq-pagebuilder/sanity-plugin-schema-default` package version is 0.1.0 as per most recent update.